Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -2244,6 +2314,23 @@ private String getShortErrorMessage(@NotNull String methodName, @Nullable String
            ", error code = " + error.getCode() + ": " + error.getMessage();
   }
 
+  private void logTookTooLongMessage(@NotNull final String methodName,
+                                     final long timeout,
+                                     @Nullable final String contextID,
```

Comment:
Looks like there's only one call where `contextID` is not null, and having it in log won't help much, so I removed this arg for the sake of simplicity.

Reply:
I went back and forth, I didn't feel strongly either way.